public/css/style.css

```diff
@@ -357,7 +357,7 @@ ul {
   z-index: 1;
   /* SATELIOT */
   /* top: var(--top-menu-height); */
-  top: 59px;
+  top: 59px; /* TODO: This will create a gap on some screen sizes since the top menu height is not fixed */
```

Comment:
I just added a comment to make this easier to find. The --top-menu-height variable is set in the various responsive.css files as the screen size changes. I think you meant to change it there, vs adding a single override here. It looks fine on a normal desktop, but if you shrink the window or use mobile, there is now a gap between the search menu and the top menu.